runtime-modules/project-token/src/lib.rs

```diff
@@ -161,11 +146,27 @@ decl_storage! { generate_storage_info
 
         /// Minimum revenue split duration constraint
         pub MinRevenueSplitDuration get(fn min_revenue_split_duration) config(): T::BlockNumber;
+
         /// Minimum revenue split time to start constraint
         pub MinRevenueSplitTimeToStart get(fn min_revenue_split_time_to_start) config(): T::BlockNumber;
 
         /// Platform fee (percentage) charged on top of each sale purchase (in JOY) and burned
         pub SalePlatformFee get(fn sale_platform_fee) config(): Permill;
+
+        /// Percentage threshold for deactivating the amm functionality
+        pub AmmDeactivationThreshold get(fn amm_deactivation_threshold) config(): Permill;
+
+        /// AMM buy transaction fee percentage
+        pub AmmBuyTxFees get(fn amm_buy_tx_fees) config(): Permill;
+
+        /// AMM sell transaction fee percentage
+        pub AmmSellTxFees get(fn amm_sell_tx_fees) config(): Permill;
+
+        /// Max patronage rate allowed
+        pub MaxYearlyPatronageRate get(fn max_yearly_patronage_rate) config(): YearlyRate = YearlyRate(Permill::from_percent(15));
+
+        /// Minimum slope parameters allowed for AMM curve
+        pub MinAmmSlopeParameter get(fn min_amm_slope_parameter) config(): TokenBalanceOf<T>;
```

Comment:
Just to be clear, all these new storage values will have value of zero (with exception of `max_yearly_patronage_rate`) after the runtime upgrade as they are not initialized at genesis.
If we need them to have some other initial value we will want a runtime migration.